.github/workflows/validate.yml

```diff
@@ -0,0 +1,44 @@
+on: [ push ]
+
+jobs:
+  build_ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+    - name: gitconfig
+      run: |
+        git config --global core.autocrlf input
+
+    - name: checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install Rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: aarch64-apple-darwin
```

Comment:
just curious why the target is target: aarch64-apple-darwin

Reply:
Good question I meant to ask @someara about that as well. It was in his original build script I cannibalized

Reply:
That's a bug... I've made a PR to fix it here https://github.com/zerotier/ZeroTierOne/pull/1975